app/tests/fixtures/lti_platform_deployment_fixture.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Note, no tests in here yet

Reply:
The fixtures are for `Lti13TestCase::test_findIssuers()`.